Metadata: display_name syntax for algorithm title (#87)
north-road/qgis-processing-r: website/pages/script-syntax.md

## Metadata

> The R script's file name is used to defined the script name and id in the R processing provider. You can ovveride these

Reviewer: ```suggestion
The R script's file name is used to define the script name and id in the R processing provider. You can override these
```

default metadata with these lines.

`##script_name=name` _script_name_ is the name of the script; under this name, it will be listed in processing toolbox.

> It is also used to defined the script id in the R processing provider.

Reviewer: ```suggestion
It is also used to define the script id in the R processing provider.
```

`##script_title=display_name` _script_title_ is the title of the script; under this title, it will be listed in processing toolbox.

> It overried the _name_ if it is defined after.

Reviewer: ```suggestion
It overrides the _name_ if it is defined after.
```
